roles/ood_user_reg_cloud/templates/ini.j2

```diff
@@ -1,23 +1,218 @@
-[uwsgi]
```

Comment:
This ini.j2 file should be added as a new file. If we want to keep the plain name ini.j2 the original file should be git rm'd and the new template added.   If we want eventual support for either gunicorn or uwsgi we can add this new ini.j2 template as gunicorn-ini.j2.  We shouldn't make this look like edits to the original ini.j2 since I assume uwsgi and gunicorn config files are not inter-changable.  They just happen to share the same name.